source/deprecations/ember-cli/v2.x.html.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Deprecations for Ember CLI v2.x
+alias: guides/deprecations/
+layout: deprecations
+---
+
+## Deprecations Added in Ember CLI 2.x
+
+### Deprecations Present in Ember CLI 2.0.0
+
+#### Migrate from Brocfile.js to ember-cli-build.js
+
+###### until: 3.0.0
+###### id: cli.brocfile
+
+Early versions of Ember CLI utilized the default build file of Broccoli: `Brocfile.js`. Over time
+we began realizing that this was not a tenable solution (we could not pass high fidelity objects
+into the build pipeline, and therefore created two instances of all addons, etc), and introduced a
+replacement for `Brocfile.js`: `ember-cli-build.js`.  The new structure allows Ember CLI to pass
+information into the function exported by `ember-cli-build.js` and avoid the issues mentioned above.
+
+The migration from `Brocfile.js` to `ember-cli-build.js` is fairly straight forward:
+
+Migrate `Brocfile.js` from:
+```js
+// Brocfile.js
+var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+var app = new EmberApp();
+module.exports = app.toTree();
+```
+
+To `ember-cli-build.js`:
+
+```js
+var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  var app = new EmberApp(defaults, {
+      // Any options
+  });
+
+  return app.toTree();
+};
+```
+
+### Deprecations Added in Ember CLI 2.2.0
+
+#### Deprecate Utility
+
+###### until: 3.0.0
+###### id: cli.ui.deprecate
+
+Using the `ember-cli/lib/utlities/deprecate` module to issue deprecations has been deprecated
+in favor of using `.writeDeprecateLine` method on the `ui` object (which is available on both
+addons and project instances).
+
+Migrate from:
+
+```js
+const deprecate = require('ember-cli/lib/utilities/deprecate');
+
+module.exports = {
+  name: 'my-addon-name',
+
+  someMethod() {
+    deprecate('this thing is deprecated!');
+  }
+}
+```
+
+To:
+
+```js
+module.exports = {
+  name: 'my-addon-name',
+
+  someMethod() {
+    this.ui.writeDeprecateLine('this thing is deprecated!');
+  }
+}
+```
+
+### Deprecations Added in Ember CLI 2.7.0
+
+#### Base URL
+
+###### until: 3.0.0
+###### id: cli.baseURL
+
+The usage of `baseURL` setting in `config/environments.js` has been deprecated in favor of using
+an explicit `rootURL`. A detailed explanation of the problem and the migration path can be found in
+[this blog post](/blog/2016/04/28/baseURL.html).
```

Comment:
Ah, will become a link to https://www.emberjs.com/blog/2016/04/28/baseURL.html when actually deployed 👍